client/src/main/java/io/confluent/kafka/schemaregistry/client/CachedSchemaRegistryClient.java

```diff
@@ -39,21 +41,43 @@
   private final Map<String, Map<Integer, Schema>> idCache;
   private final Map<String, Map<Schema, Integer>> versionCache;
 
-  public CachedSchemaRegistryClient(String baseUrl, int identityMapCapacity) {
-    this(new RestService(baseUrl), identityMapCapacity);
+  public CachedSchemaRegistryClient(
```

Comment:
We need to be careful about API compatibility here. This class is considered public API as it is the standard Java API for interacting w/ the schema registry.

Reply:
I didn't know that we documented it as a public API. Anyways, i will make it backward compatible.

Reply:
There's some stuff in SR that we're probably not clear about, I'm not sure it is strictly documented. However, in practice this is the std Java API, so it effectively has to be public API.